Swiftz/ArrayExt.swift

```diff
@@ -273,13 +311,13 @@ extension Array {
 		}
 	}
 
-	/// Maps a predicate over a list.  For the result to be true, the predicate must be satisfied at
+	/// Maps a predicate over a list.  For the Array to be true, the predicate must be satisfied at
```

Comment:
You may want to reconsider this change, and some of the ones above and below it.  :þ